dynamic_programming/maximum_circular_subarray.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+* @file
+* @brief C++ program for maximum contiguous circular sum problem using Kadane's Algorithm [https://en.wikipedia.org/wiki/Maximum_subarray_problem]
+* @details
+* The idea is to modify Kadane’s algorithm to find a minimum contiguous subarray sum and the maximum contiguous subarray sum,
+* then check for the maximum value between the max_value and the value left after subtracting min_value from the total sum.
+* Geeks For Geeks link [https://www.geeksforgeeks.org/maximum-contiguous-circular-sum/]
+*/
+
+#include <cassert>   
+#include <iostream>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```